File: sandbox/fixtures/rules/go/GO_S172_println/src/smelly.go
// Smelly: fmt.Println in library code
package main

import "fmt"

func processData(data []int) []int {
	fmt.Println("Processing data:", data)
	result := make([]int, len(data))
	for i, v := range data {
		result[i] = v * 2
	}
	return result
}

File: sandbox/fixtures/rules/python/PY_S138_long_func/src/smelly.py
# Smelly: Long function
def process_items(items):
    result = []
    for item in items:
        if item > 0:
            result.append(item * 2)
        elif item == 0:
            result.append(0)
        else:
            result.append(item + 10)
    return result
